runtime/etc/src/mli/client_runtime.c

```diff
@@ -183,5 +186,13 @@ void chpl_library_finalize(void) {
   return;
 }
 
+// TODO: proper implementation once opaque arrays are supported in multilocale
+// interoperability.
+void cleanupOpaqueArray(chpl_opaque_array * arr) {
+  printf("Error: opaque arrays not supported for multilocale libraries");
```

Comment:
I would prefer a `chpl_mli_sdebugf` here just so we know the client was emitting the error in debug mode.

Reply:
Wouldn't that prevent the error from turning up when `--library-ml-debug` is not thrown?